datashuttle/datashuttle_class.py

```diff
@@ -1079,19 +1079,21 @@ def get_next_sub(
         local_only: bool = False,
     ) -> str:
         """
-        Convenience function for get_next_sub_or_ses
+        Convenience function for `get_next_sub_or_ses`
         to find the next subject number.
 
         Parameters
         ----------
+        top_level_folder : TopLevelFolder
+            The top-level folder structure where subject data is organized.
 
-        return_with_prefix : bool
-            If `True`, return with the "sub-" prefix.
+        return_with_prefix : bool, optional
+            If `True`, return the subject with the "sub-" prefix. Defaults to `True`.
```

Comment:
Here further to [this comment](https://github.com/neuroinformatics-unit/datashuttle/issues/369#issuecomment-2156680003) and (just added) #483, we will remove type hints and defaults from the doctrings. So I think we can revert the changes that the default (but feel free to keep optional and these can all be removed together). thanks!